omod-1.11/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_11/DrugIngredientResource1_11.java

```diff
@@ -80,6 +81,21 @@ public DelegatingResourceDescription getUpdatableProperties() {
 		return getCreatableProperties();
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
No it's not due to lack of time. I had enough time :) I've been tracking undocumented resources by validating the swagger. Some how they were missed. I'll check and fix those.